src/Form/DataTransformer/ProviderDataTransformer.php

```diff
@@ -99,6 +104,10 @@ public function reverseTransform($media)
             $newMedia->setContext($this->options['context']);
         }
 
+        if (null !== $this->categoryManager) {
```

Comment:
Why do we need an `CategoryManagerInterface` instance when we only use it for a null check and ignore it afterwards completly?

Reply:
This is dangerous, we dont want to add a require dependency with ClassificationBundle again